dist/tools/testrunner/testrunner.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+#               2014 Martine Lenders <mlenders@inf.fu-berlin.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os, signal, sys, subprocess
+from pexpect import spawnu, TIMEOUT, EOF
+
+def run(testfunc, timeout=5):
+    env = os.environ.copy()
+    child = spawnu("make term", env=env, timeout=timeout)
+    child.logfile = sys.stdout
```

Comment:
This should be made optional (output is nice for debugging but from a certain stability on it should not be needed, just the result).